runtime/fdlimits/fdlimits.go

```diff
@@ -20,6 +22,6 @@ func SetMaxFdLimits() error {
 	if err != nil {
 		return err
 	}
-	log.Infof("Raised fd limit to %d from %d", raisedVal, curr)
+	log.Infof("Updated file descriptor limit to %d from %d", raisedVal, curr)
```

Comment:
Could we push this to debug? I notice it on many commands and it doesn't seem like meaningful information.

Reply:
Good point!